lib/SMT/SMTParser.cpp

```diff
@@ -9,17 +9,17 @@
 
 #include "SMTParser.h"
 
-#include "klee/ExprBuilder.h"
-#include "klee/Solver.h"
-#include "klee/Constraints.h"
-#include "expr/Parser.h"
-
 #include <fstream>
 #include <string>
 #include <sstream>
 #include <cassert>
 #include <stack>
 
+#include "../../include/klee/Expr/expr/Parser.h"
```

Comment:
No relative includes and the `klee/*` includes should come before the system ones.